mozci/console/commands/build_regressions.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
```

Comment:
Let's move all the functions from this file into push.py, before https://github.com/mozilla/mozci/blob/917898bfc6b55ee6d4a3b207ef4d6c014e3c39b8/mozci/push.py#L1128